ledger-service/http-json/src/it/scala/http/AuthorizationTest.scala

```diff
@@ -43,17 +38,18 @@ final class AuthorizationTest
   implicit val aesf: ExecutionSequencerFactory = new AkkaExecutionSequencerPool(testId)(asys)
   implicit val ec: ExecutionContext = asys.dispatcher
 
-  private val publicToken = "public"
-  private val emptyToken = "empty"
+  private val publicTokenValue = "public"
+  private val emptyTokenValue = "empty"
+
   private val mockedAuthService = Option(AuthServiceStatic {
-    case `publicToken` => ClaimSet.Claims.Empty.copy(claims = Seq[Claim](ClaimPublic))
-    case `emptyToken` => ClaimSet.Unauthenticated
+    case `publicTokenValue` => ClaimSet.Claims.Empty.copy(claims = Seq[Claim](ClaimPublic))
+    case `emptyTokenValue` => ClaimSet.Unauthenticated
   })
 
   private val accessTokenFile = Files.createTempFile("Extractor", "AuthSpec")
 
   override def authService = mockedAuthService
-  override def packageFiles = List(dar)
+  override def packageFiles = List()
```

Comment:
Why is an empty list ok here?

Reply:
This test does not need any package to be uploaded to be functional.

Reply:
As you see - CI is ok with not having this file, so this file is not needed